packages/imperative/src/config/cmd/init/init.handler.ts

```diff
@@ -88,6 +143,28 @@ export default class InitHandler implements ICommandHandler {
         config.api.layers.merge(newConfig);
     }
 
+    /**
+     * Do a dry run of creating JSON template for config.
+     * Also create a schema file in the same folder alongside the config.
+     * @param config Config object to be populated
+     * @param user If true, properties will be left empty for user config
+     */
+    private async initForDryRun(config: Config, user: boolean): Promise<any> {
+        // Build the schema and write it to disk
+        const schema = ConfigSchema.buildSchema(ImperativeConfig.instance.loadedConfig.profiles);
+        config.setSchema(schema);
+
+        const opts: IConfigBuilderOpts = {};
+        if (!user) {
+            opts.populateProperties = true;
+            opts.getSecureValue = this.promptForProp.bind(this);
```

Comment:
No need to change anything here, just an observation 🙂 

The inclusion of prompting when `--dry-run` flag is passed struck me as odd at first when reviewing this PR, but it seems kind of necessary in order to show a complete diff of the changes.